--- a/sdk/pkcs11/samples/PKIExtensions/Makefile	2019-09-16 09:42:40.000000000 -0700
+++ b/sdk/pkcs11/samples/PKIExtensions/Makefile	2020-02-13 07:43:17.682581000 -0800
@@ -12,6 +12,8 @@
 ARCH_CFLAGS_BSD32 = -m32 -march=i486 -mtune=i486
 ARCH_CFLAGS_BSD64 = -m64 -march=nocona -mtune=generic
 
+ARCH_CFLAGS_LINUX_ARM =-mtune=cortex-a7 -mcpu=cortex-a7 -mfpu=vfpv4-d16
+
 ARCH_LDFLAGS = -ldl -lpthread
 ARCH_LDFLAGS_BSD = -lpthread
 
@@ -56,6 +58,12 @@
 
 OUTDIR = ./../out/${PLATFORM}-${CONFIGURATION}
 
+ifneq (, $(findstring arm, ${ARCH}))
+	ARCH_CFLAGS = ${ARCH_CFLAGS_LINUX_ARM}
+	PLATFORM = linux_glibc-armv7hf
+	OUTDIR = ./../../pkcs11ecp-samples-${PLATFORM}
+endif
+
 CFLAGS += -x c
 CFLAGS += -Werror -Wall -Wextra -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function
 CFLAGS += -I./../../include -I./../include
