--- a/sdk/openssl/rtengine/samples/Makefile	2019-09-16 09:42:40.000000000 -0700
+++ b/sdk/openssl/rtengine/samples/Makefile	2020-02-14 06:27:05.948472627 -0800
@@ -4,34 +4,14 @@
 
 DIR = $(dir $(lastword $(MAKEFILE_LIST)))
 
-ifneq (, $(findstring darwin, $(shell $(CC) -dumpmachine)))
-	AUTO_TARGET_OS = osx
-else
-	AUTO_TARGET_OS = linux_glibc
-endif
-
-ifndef TARGET_OS
-	TARGET_OS = $(AUTO_TARGET_OS)
-endif
+ARCH_CFLAGS_LINUX_ARM =-mtune=cortex-a7 -mcpu=cortex-a7 -mfpu=vfpv4-d16
 
 
-ifeq ($(TARGET_OS), osx)
-	AUTO_TARGET_ARCH = x86_64
-else ifneq (, $(findstring x86_64, $(shell ${CC} -dumpmachine)))
-	AUTO_TARGET_ARCH = x86_64
-else
-	AUTO_TARGET_ARCH = x86
-endif
-
-ifndef TARGET_ARCH
-	TARGET_ARCH = $(AUTO_TARGET_ARCH)
-endif
+CFLAGS = ${ARCH_CFLAGS_LINUX_ARM}
+TARGET_ARCH = armv7hf
+TARGET_OS = linux_glibc
+PLATFORM_DIR = linux_glibc-armv7hf
 
-ifeq ($(TARGET_OS), osx)
-	PLATFORM_DIR = macos-x86_64
-else
-	PLATFORM_DIR = $(TARGET_OS)-$(TARGET_ARCH)
-endif
 
 BIN_DIR = $(DIR)../bin/$(PLATFORM_DIR)
 OPENSSL_DIR = $(DIR)../..
@@ -57,14 +37,6 @@
 endif
 
 
-ifeq ($(TARGET_OS), osx)
-	COMMON_FLAGS += -arch x86_64
-else ifeq ($(TARGET_ARCH), x86_64)
-	COMMON_FLAGS += -m64
-else
-	COMMON_FLAGS += -m32
-endif
-
 CFLAGS_H = $(CFLAGS) -DHARDWARE_KEYS
 LDFLAGS_H = $(LDFLAGS) -ldl
 
